### ISSUE TITLE
DiscordLink: Copy Code Via Chat

### ISSUE THREAD

**Issue by skrrtn** [EssentialsX/Essentials#5542]
### Feature description

The ability to copy the Discord Link code via Minecraft chat by clicking on it.

### How the feature is useful

Would cut down on the time it takes to link an account. Makes it easier for the player.

**Comment by Evidentsinger14** [comment]
This should be possible once Adventure is merged into EssentialsX. Keep an eye on [this issue](https://github.com/EssentialsX/Essentials/issues/2391) for when that happens!
